scripts/feature_tests/upgrade/upgrade.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -x
+
+# Remove old test result
+rm -rf /tmp/"$(date +%Y.%m.%d_upgrade.result.txt)"
+
+M3_DIR="$(dirname "$(readlink -f "${0}")")/../../.."
+
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${M3_DIR}/lib/common.sh"
+
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${M3_DIR}/lib/network.sh"
+
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${M3_DIR}/lib/images.sh"
+
+# Fetch the upgrade tests from airship-dev-tools
+pushd "${M3_DIR}"/scripts/feature_tests/upgrade || exit
+cd /tmp || exit
+git clone https://github.com/Nordix/airship-dev-tools.git
+popd || exit
+
+cp -r /tmp/airship-dev-tools/upgrade_tests "${M3_DIR}"/scripts/feature_tests/upgrade
```

Comment:
are you replacing the existing tests from Metal3-dev-env by the ones in airship-dev-tools ?